Analyses/test/bfieldPlotter.fcl

```diff
@@ -0,0 +1,32 @@
+# -*- mode:tcl -*-
+# Example of running BFieldPlotter
+#
+
+#include "JobConfig/primary/flateminus.fcl"
+process_name : BFieldPlotter
+
+physics.analyzers.bfieldplotter : {
+    module_type : BFieldPlotter
+    plane       :  "x"
+    planeValue  :  -10000.
+    axisOneMin  : -8050.  # -8050. #-10500.
+    axisOneMax  :  8050.  #  8050. # 10500.
+    axisTwoMin  : -25000. # -8050. # -25000.
+    axisTwoMax  :  45000. #  8050. #  45000.
+    mapBinSize  :  10.
+}
+
+physics.e1             : [bfieldplotter]
+physics.end_paths      : [e1]
+physics.p1             : [generate, g4run]
+physics.trigger_paths  : [p1]
+
+services.TFileService.fileName: "nts.owner.bfieldplotter.x_-10000.sequencer.root"
+outputs.Output.fileName       : "dig.owner.bfieldplotter.version.sequencer.art" #shouldn't be generated
+
+#Use file with altered field maps
+services.GeometryService.inputFile : "Analyses/test/geom_bfieldplotter.txt"
+
+#Don't need full physics to map the field
+physics.producers.g4run.physics.physicsListName : "Minimal" 
+
```

Comment:
This is not needed since you do not need G4.

Reply:
I've removed this and g4run being used